modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/util/ShowPartialResultsAlert.tsx

```diff
@@ -38,9 +42,7 @@ const ShowPartialResultsAlert: React.FC<IProps> = ({
 				onClose={dismissible ? () => setDismissed(true) : undefined}
 				title="Info"
 			>
-				{Liferay.Language.get(
-					'your-responses-will-be-visible-to-all-form-respondents'
-				)}
+				{Liferay.Language.get(showPartialResultsMessage)}
```

Comment:
Also change this line to:

```suggestion
				{showPartialResultsMessage}
```